swap/swap_test.go

```diff
@@ -1083,3 +1084,29 @@ func (d *dummyMsgRW) ReadMsg() (p2p.Msg, error) {
 func (d *dummyMsgRW) WriteMsg(msg p2p.Msg) error {
 	return nil
 }
+
+func TestFileLogger(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in short mode.")
+	}
+
+	handler, err := log.FileHandler("/home/git/go/src/github.com/ethersphere/swarm/testlog/test.log", log.TerminalFormat(true))
+	if err != nil {
+		t.Fatal(err)
+	}
+	log.Root().SetHandler(log.LvlFilterHandler(log.Lvl(*loglevel), handler))
+	log.Error("Test")
```

Comment:
After this, we should test whether "Test" is indeed in the test log, not?

Reply:
Yes exactly, I wanted to get some feedback on the logging implementation, before doing all the tests.
But as you stated, the log should be compared to verify the intended behavior.